samples/Common.c

```diff
@@ -698,8 +736,11 @@ STATUS freeSampleConfiguration(PSampleConfiguration* ppSampleConfiguration)
     pSampleConfiguration = *ppSampleConfiguration;
 
     CHK(pSampleConfiguration != NULL, retStatus);
-
     for (i = 0; i < pSampleConfiguration->streamingSessionCount; ++i) {
+        // Run export LOG_STATS=TRUE to see logs
```

Comment:
I'm a little bit confused here. Why do we need `LOG_STATS=TRUE`? If users are not interested, can't they just not call `getIceServerStats`?

Reply:
I have added it to keep it consistent with the way I set up in Metrics.c. I use an export flag to control logging of all stats to avoid any unnecessary spew. And yes, if users are not interested they can avoid calling. In this example, I am just logging the stats. But, users can consume the information any way they want. So logging and extracting metrics can be two different things.

Reply:
I see what you mean. That makes sense.  Since it's a part of `rtcPeerConnectionGetMetrics`, maybe move the comment to be above `rtcPeerConnectionGetMetrics` call?